nestkernel/mpi_manager.cpp

```diff
@@ -713,6 +713,30 @@ nest::MPIManager::grng_synchrony( unsigned long process_rnd_number )
   return true;
 }
 
+// wfr_synchrony: called at the beginning of each simulate
+bool
+nest::MPIManager::wfr_synchrony( bool any_node_uses_wfr )
```

Comment:
I think we should be more abstract at the level of `MPIManage` and choose a method name that expresses the logical operation, not the semantic purpose in one special case. If I understand the code and context right, this method takes a single bool, exchanges with all other processes, and returns `true` if one or more processes provide `true`. In NumPy-parlance, this is `any()`.

I am wondering about two things:
1. What would be a good name? I fear `any()` is a bit too short.
1. Is there an MPI function that provides this type of aggregation built-in?

I realize that I wrote the similarly problematic `grng_synchrony()` myself ...